src/magento-payments/payoutreports.md

```diff
@@ -0,0 +1,133 @@
+Last 90 days | Available from the Payout dates selector, this report shows information for the last 90 days.
+Year to date | Available from the Payout dates selector, this report shows information for the the entire year to date.
+Custom range | Available from the Payout dates selector, this report can be filtered to show a custom date range.
+{:style="table-layout:auto;"}
+
+## Column descriptions
+
+Payout reports include the following information.
+
+Column | Description
+------------ | --------------------
+Provider | Payment provider (PayPal)
+Provider trans | PayPal transaction ID
+Trans date | Date and time transaction was initiated
+Type | Transaction type (PAYMENT, AUTH, BONUS, CHARGEBACK, CORRECTION, CURRENCY_CONVERSATION, DEPOSIT, DISBURSEMENT, DISPUTE, FEES, HOLD, HOLD_RELEASE, INCENTIVES, OTHERS, RECOUP, REFUND, REVERSAL, WITHDRAWAL) <br> <br>See [Transaction types](#transaction-types) for more information.
+Status | Current status of the transaction (SUCCESS, DENIED, PENDING)
+Code | Transaction code that indicates either Credit or Debit
+Reference ID | Original PayPal transaction ID for which this event is related
+Invoice | Invoice ID (one per order) of the transaction
+Magento order | Magento order ID
+Magento trans | Magento transaction ID
+Pay method | Credit card type (BANK, PAYPAL, CREDIT_CARD) and associated card provider (Visa, MasterCard, etc.)
+Trans amt | Amount of the transaction
+Cur (trans amt) | Currency unit for transaction amount
+Pending | Amount yet to be disbursed
+Cur (pending) | Currency unit for the pending amount
+Seller amt | Amount of funds transferred to or from a customer; Funds moving out of the seller account show a dash (-) prefix.
+Cur (seller amt) | Currency unit for the seller amount
+Partner fee | Partner fees associated with the transaction; Funds moving out of the partner fee account show a dash (-) prefix.
+Cur (partner fee) | Currency unit for the partner fee
+Prov feed | PayPal fees associated with the transaction; Funds moving out of the PayPal fee account show a dash (-) prefix.
+Cur (prov fee) | Currency unit for the PayPal fee
+Fee% | Percentage of the transaction amount charged as a fee
+Fixed fee | Fixed PayPal fee amount
+Chbk fee | Chargeback fee associated with the transaction; A dash (-) prefix indicates the chargeback fee was reversed.
+Cur (Chbk fee) | Currency unit for the chargeback fee
+Hold amt | Amount put on hold or released from hold; A dash (-) prefix indicates on hold funds are being released.
+Cur (hold amt) | Currency unit for the hold amount
+Recoup amt | Amount recouped from the recoup account; Funds moving out of the recoup account show a dash (-) prefix.
+Cur (recoup amt) | Currency unit for the recoup amount
+{:style="table-layout:auto;"}
+
+## Transaction types
+
+These transaction types may be noted in Payout reports.
+
+Report | Description
+------------ | --------------------
+PAYMENT | Money moved between a buyer and a seller for an order
```

Comment:
The All Caps thing is another question. Even when used in the UI, we have been steering away from it in docs.  But I am thinking you might have these as placeholders for now :)

Reply:
I am representing it as it looks in the UI, correct. Given these are statuses, and those are typically pulled from code, I formatted them as per how they view for the user to keep things clear.